TheMovieDbTests/DetailViewModelTests.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  DetailViewModelTests.swift
+//  TheMovieDbTests
+//
+//  Created by Ricardo Ramirez on 18/11/21.
+//
+
+import XCTest
+@testable import TheMovieDb
+
+class DetailViewModelTests: XCTestCase {
```

Comment:
In general terms how did you feel making tests?

Reply:
I was a little doubtful at the start but after creating some tests it becomes easier